axon-server-connector/src/test/java/org/axonframework/axonserver/connector/processor/EventProcessorControlServiceTest.java

```diff
@@ -27,15 +28,18 @@
 
 class EventProcessorControlServiceTest {
```

Comment:
I'm missing tests for checking whether instruction result is sent after split/merge.